data-serving/scripts/export-data/country_export.sh

```diff
@@ -12,10 +12,10 @@ BUCKETS="${SCRATCH}/buckets.json"
 trap 'rm -rf "$SCRATCH"' EXIT  # Cleanup before exit
 
 FORMAT="${FORMAT:-csv,tsv,json}"
-QUERY="{\"list\": true, \"location.country\": \"$COUNTRY\"}"
+QUERY="{\"location.countryISO3\": \"$COUNTRY\"}"
```

Comment:
data should be approved by a senior curator?

Reply:
I completely forgot about this, good call, thanks.

Reply:
To be resolved in another PR?